common/config/rush/pnpm-lock.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
   '@rush-temp/storage-queue': 'file:projects/storage-queue.tgz'
   '@rush-temp/tables': 'file:projects/tables.tgz'
   '@rush-temp/template': 'file:projects/template.tgz'
+  '@rush-temp/templates': 'file:projects/templates.tgz'
```

Comment:
What is this new templates package for?

Reply:
This is for testing purposes. If there is a package with a plural name

Reply:
Will be removing all commits except first after verification

Reply:
Can you link to the issue you are trying to fix so I can see more context on what you are trying to fix?

Reply:
When we try to release core-http today, the pipeline failed, because the scripts to stage artifacts also copied `azure-core-https-1.0.0.preview.1.tgz` to the staging folder. 

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=442574&view=logs&j=eb742a6b-6c25-5990-0095-a3251c7e31a3&t=c948a392-34ba-521f-13c3-bcf399a4a0fb

Reply:
@KarishmaGhiya so is this issue a temporary fix for https://github.com/Azure/azure-sdk-for-js/issues/9312? My hope is we can eliminate this class of problems completely once we tackle that issue.

Reply:
I think so yeah. we will be able to eliminate these kind of issues with #9312